.gitlab-ci.yml

```diff
@@ -22,8 +22,13 @@ build_qa:
     - $(aws ecr get-login --no-include-email --region $AWS_DEFAULT_REGION)
     - git clone https://${GITHUB_TOKEN}:x-oauth-basic@github.com/meedan/configurator ./configurator
     - d=configurator/check/qa/check-api/; for f in $(find $d -type f); do cp "$f" "${f/$d/}"; done
-    - docker build  -f Dockerfile.production.aws -t "$ECR_API_BASE_URL/qa/check/api:$CI_COMMIT_SHA"  .
-    - docker push "$ECR_API_BASE_URL/qa/check/api:$CI_COMMIT_SHA"
+    - > 
+      if docker image list | grep -q "$CI_COMMIT_SHA"; then 
```

Comment:
This will not work for `gitlab-ci` because the image does not ever exist on the host. You need to query the registry to see if it's there.